account-kit/logging/src/index.ts

```diff
@@ -9,12 +7,6 @@ export type * from "./types.js";
 export function createLogger<
   Schema extends EventsSchema
 >(): EventLogger<Schema> {
-  if (WRITE_KEY == null) {
-    // If we don't have a write key, we don't want to log anything
-    // This is useful for dev so we don't log dev metrics
-    return noopLogger;
-  }
```

Comment:
I'm making this a responsibility of the server / client loggers in case their implementations are different and they don't care about WRITE_KEY or care about something else